sw/device/silicon_creator/lib/sigverify_tests/sigverify_testvectors.h

```diff
@@ -0,0 +1,221 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// AUTOGENERATED. Do not edit this file by hand.
```

Comment:
I actually prefer adding this file to the repo but we should have a CI check to ensure that it is in sync with the json file. @jadephilipoom Please feel free to create an issue and address in a separate PR.

Reply:
Created https://github.com/lowRISC/opentitan/issues/9865

The reason for committing the autogenerated file is so that it's easy to run the default test vectors without going through the extra steps of generating the header (and so it's easier to see what tests are running without understanding the whole test setup, which has some added complexity in order to handle large test vector sets from external sources). But a CI check would indeed be a good idea here.